source/docs/guides/EMV_3D_secure.md

```diff
@@ -564,6 +559,338 @@ trnApproved=1
+    2Q2LWFjNDgtZGVmZTczZDJhMWVmIn0"
+}
+```
+
+### GET /[threeDS_session_data]
+
+Use this endpoint to fetch information about previous 3DSv2 sessions.
+		Responses from this endpoint will always take the following format:
+
+|Parameter|Data Type|Description|
+|---:|---:|---|
+|threeDS_session_data|string|Unique identifier for this 3DSv2 session.|
+|transaction_id|Numeric|Identifier for the transaction in your transaction reports which the 3DSv2 session is associated with|
+|amount|Currency|Value of the transaction in dollars and cents (or chosen currency equivalent)|
+|card.bin|String|First six digits of card number|
+|card.last_four|String|Last four digits of card number|
+|card.expiry_month|String|Two-digit expiration month of card|
+|card.expiry_year|String|Four-digit expiration year of card|
+|protocolVersion|String|The type/version of the 3DSv2 session|
+|dsTransactionId|String|The directory server transaction identifier|
+|xid|Numeric|The 3DS transaction identifier|
+|eci|Numeric|The electoronic commerce indicator|
+|cavv|String|The cardholder authentication verification value|
+|flow_type|String|Indicates the 3DS flow completed for this transaction set to 'F' for Frictionless, and 'C' for Challenge|
+|status|String|Enum string representing the status of the 3DSv2 session|
+|error|String|If the request resulted in an error this will contain the emun error indentifier|
+
+#### Sample GET Request
+
+```shell
+curl --location --request GET 'https://api.na.bambora.com/v1/EMV3DS/MDBiYmI5NTYtOTEwNy0A5Y2FlZDMxY2Vh' \
+--header 'Authorization: Passcode MzkwOTgwMDAwOmJhbWJvcmE=' \
+--header 'Content-Type: application/json'
+```
+
+#### Sample GET Response
+
+```shell
+{
+    "threeDS_session_data": "MDBiYmI5NTYtOTEwNy00MzU1LTg0YmUtMTA5Y2FlZDMxY2Vh",
+    "amount": 55.66,
+    "card": {
+        "bin": "456735",
+        "last_four": "7977",
+        "expiry_month": "05",
+        "expiry_year": "2019"
+    },
+    "flow_type": "F",
+    "status": "Success",
+    "authorization": {
+        "eci": "5",
+        "cavv": "AAABBEg0VhI0VniQEjRWAAAAAAA=",
+        "xid": 12093654972,
+        "ds_transaction_id": "63AEEFE8-BE1F-41FA-AD52-4B5E2E16671D",
+        "protocol_version": "2.2"
+    },
+    "error": null,
+    "created_datetime_utc": "2021-12-14T02:11:50.45"
+}
+```
+
+### Processing a Payment Using a 3DS Sesstion Data Token
+
+A credit card that has been successfully authenticated through the EMV 3DS API can be later referenced to complete a payment without needing to resubmit the credit card number and expiry. The value of the 'threeDS_session_data' parameter returned in the authentication response can be used a card number token when calling the Payment API to charge the card.
+
+In the payment request we must set the 'payment_method' to a value of '3d_secure' then pass a '3d_secure' object containing the 'threeDS_session_data' as the card token and indicate if 'complete' should be true or false.
```

Comment:
Comma needed after the first '3d_secure'.
Probably one after 'the card token' as well, but not as critical